site/_data/tweets.js

```diff
@@ -134,19 +127,11 @@ const formatEntities = tweet => {
  * @return {Promise<TwitterTweet[]>}
  */
 module.exports = async () => {
-  let tweets = process.env.CI
-    ? require('./tweets-sample.json')
-    : await CacheAsset(url, {
-        duration: '1h',
-        type: 'json',
-      }).catch(e => {
-        console.warn(e);
-        if (process.env.NODE_ENV === 'production') {
-          return undefined;
-        } else {
-          return require('./tweets-sample.json');
-        }
-      });
+  // We have to cast this to unknown first as Typescript gets very literal about the underlying type
+  // of the JSON.
+  let tweets = /** @type {TwitterTweet[]} */ (
+    /** @type {unknown} */ (require('../../external/data/tweets.json'))
```

Comment:
Maybe we should use the cache tool here to point to the external tweets if there isn't a local one.

Reply:
I have mixed feelings on this but I'm happy to be disagreed with. I think there's a big advantage in _not_ having to have conditionals here—the code always just pulls _something_ in.

But I've added an eslint rule to allow this to be missing and that's awkward too.

WDYT?